src/api/spec/controllers/webui/comments_controller_spec.rb

```diff
@@ -55,6 +53,47 @@
       end
     end
 
+    context 'with a commentable that does not exist' do
+      context 'of a project' do
+        let(:commentable) { project }
+
+        before do
+          params = { comment: { body: "This #{commentable.model_name.singular} is AWESOME!" },
```

Comment:
I don't think `singular` is needed here. I suggest simply typing `project` instead of having some code which is pretty much like the implementation.

```suggestion
          params = { comment: { body: "This #{commentable.model_name} is AWESOME!" },
```

Reply:
True, I unpacked a shared_context and forgot about that line...